kivy/graphics/context_instructions.pyx

```diff
@@ -764,9 +764,25 @@ cdef class Scale(Transform):
     .. deprecated:: 1.6.0
         Deprecated single scale property in favor of x, y, z, xyz axis
         independent scaled factors.
```

Comment:
Considering https://github.com/kivy/kivy/issues/8177, I guess we can just remove the single scale property.